packages/eas-cli/src/submissions/ios/AppProduce.ts

```diff
@@ -1,14 +1,11 @@
 import { App, RequestContext, Session, User } from '@expo/apple-utils/build';
 import { getConfig } from '@expo/config';
-import wordwrap from 'wordwrap';
 
 import { getBundleIdentifier } from '../../build/ios/bundleIdentifer';
 import { authenticateAsync, getRequestContext } from '../../credentials/ios/appstore/authenticate';
-import { USE_APPLE_UTILS } from '../../credentials/ios/appstore/experimental';
 import log from '../../log';
 import { promptAsync } from '../../prompts';
 import { IosSubmissionContext } from '../types';
-import { runFastlaneAsync, travelingFastlane } from '../utils/fastlane';
 import { sanitizeLanguage } from './utils/language';
 
 interface ProduceOptions {
```

Comment:
Maybe sth like
```suggestion
 interface AppCreationOptions {
```
since `runProduceAsync()` is also renamed this way